components/tags/skin.css

```diff
@@ -10,62 +10,106 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+:root {
+  --spectrum-tag-background-color-error-selected: var(--spectrum-global-color-red-500);
+  --spectrum-tag-background-color-error-selected-hover: var(--spectrum-global-color-red-600);
+}
+
 .spectrum-Tags-item {
   color: var(--spectrum-tag-text-color);
   background-color: var(--spectrum-tag-background-color);
   border-color: var(--spectrum-tag-border-color);
 
-  .spectrum-ClearButton {
-    color: var(--spectrum-tag-deletable-icon-color);
+  .spectrum-Tags-itemIcon {
+    color: var(--spectrum-tag-icon-color);
   }
 
   &:hover {
     background-color: var(--spectrum-tag-background-color-hover);
     color: var(--spectrum-tag-text-color-hover);
     border-color: var(--spectrum-tag-border-color-hover);
 
-    .spectrum-ClearButton {
-      color: var(--spectrum-tag-deletable-icon-color-hover);
+    .spectrum-Tags-itemIcon {
+      color: var(--spectrum-tag-icon-color-hover);
     }
   }
 
-  &:focus-ring {
+  &.focus-ring {
     background-color: var(--spectrum-tag-background-color-key-focus);
     color: var(--spectrum-tag-text-color-key-focus);
     border-color: var(--spectrum-tag-border-color-key-focus);
     box-shadow: 0 0 0 1px var(--spectrum-tag-border-color-key-focus);
 
-    .spectrum-ClearButton {
-      color: var(--spectrum-tag-deletable-icon-color-key-focus);
+    .spectrum-Tags-itemIcon {
+      color: var(--spectrum-tag-icon-color-key-focus);
     }
   }
 
-  &.is-invalid {
-    color: var(--spectrum-tag-text-color-error);
-    border-color: var(--spectrum-tag-border-color-error);
+  &.is-selected {
+    background-color: var(--spectrum-tag-background-color-selected);
+    color: var(--spectrum-tag-text-color-selected);
+    border-color: var(--spectrum-tag-border-color-selected);
+    
+    .spectrum-Tags-itemIcon {
+      color: var(--spectrum-tag-icon-color-selected);
+    }
 
-    .spectrum-ClearButton {
-      color: var(--spectrum-tag-icon-color-error);
+    &:hover {
+      background-color: var(--spectrum-tag-background-color-selected-hover);
+    }
+
+    &.focus-ring {
+      box-shadow: 0 0 0 2px var(--spectrum-tag-border-color-selected-key-focus);
+      border-color: var(--spectrum-tag-text-color-selected-key-focus);
+    }
+
+    &.is-invalid {
+      border-color: var(--spectrum-tag-border-color-error-selected);
+      background-color: var(--spectrum-tag-background-color-error-selected);
+      .spectrum-Tags-itemIcon,
+      .spectrum-Tags-itemLabel,
+      .spectrum-Tags-itemClearButton {
+        color: var(--spectrum-tag-text-color-error-selected);
+      }
+      &.focus-ring {
+        box-shadow: 0 0 0 2px var(--spectrum-tag-border-color-error-selected-key-focus);
```

Comment:
I think DNA beta 6.0 has that focus ring variable:
https://git.corp.adobe.com/Spectrum/spectrum-dna/tree/6.0-beta

`        "spectrum-alias-focus-ring-size": "2px",`  Not seeing one for tags there yet.